File: python/VeriPaketi.py
class VeriPaketi(KuantumNesnesi):
    def __init__(self, id, stabilite, t_seviyesi):
        super().__init__(id, stabilite, t_seviyesi)
    
    def analiz_et(self):
        yeni_deger = self.stabilite - 5

        if yeni_deger <= 0:
            raise KuantumCokusException(f"SİSTEM ÇÖKTÜ! ID: {self.id}")
        
        self.stabilite = yeni_deger
        print("Veri degeri okundu")

File: java/VeriPaketi.java
public class VeriPaketi extends KuantumNesnesi {
    public VeriPaketi(String id, int stabilite, int tSeviyesi){
        super(id, stabilite, tSeviyesi);
    }

    @Override
    public void analizEt() throws KuantumCokusException{
        int yeniDeger = getStabilite() - 5; 
        if (yeniDeger <= 0) {
            throw new KuantumCokusException(getNesneID());
        }
        setStabilite(yeniDeger);
        System.out.println("Veri icerigi okundu.");
    }
}
